staging/src/k8s.io/apiserver/pkg/endpoints/handlers/create.go

```diff
@@ -130,7 +130,7 @@ func createHandler(r rest.NamedCreater, scope *RequestScope, admit admission.Int
 
 		ae := request.AuditEventFrom(ctx)
 		admit = admission.WithAudit(admit, ae)
-		audit.LogRequestObject(ae, obj, scope.Resource, scope.Subresource, scope.Serializer)
+		audit.LogRequestObject(req, obj, scope.Resource, scope.Subresource, scope.Serializer)
```

Comment:
Since you have it here, why not pass ctx directly?

Reply:
Done.